ansible_collections/arista/avd/molecule/eos_designs_unit_tests/documentation/devices/core-1-isis-sr-ldp.md

```diff
@@ -129,6 +129,8 @@ vlan internal order ascending range 1006 1199
 | Ethernet4 | P2P_LINK_TO_core-2-ospf-ldp_Ethernet4 | routed | - | - | default | 1500 | false | - | - | - | - |
 | Ethernet5 | P2P_LINK_TO_core-2-ospf-ldp_Ethernet5 | routed | - | - | default | 1500 | false | - | - | - | - |
 | Ethernet6 | P2P_LINK_TO_core-2-ospf-ldp_Ethernet6 | routed | - | - | default | 1602 | false | - | - | - | - |
+| Ethernet12 | P2P_LINK_TO_core-2-ospf-ldp_Port-Channel12 | *routed | 12 | *- | **default | *1500 | *false | **- | *- | **- | **- |
```

Comment:
these changes are caused by a bugfix i did while refactoring some of the template code. (Checking for ipv6_enabled on port_channel_interface)